config/helm/kustomization.yaml

```diff
@@ -1,2 +1,6 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: "{{ .Release.Namespace }}"
 resources:
 - ../default
```

Comment:
Could you not just delete the namespace in this kustomization rather than changing everything else to suit this ones needs?

```
patches:
  - patch: |-
      $patch: delete
      apiVersion: v1
      kind: Namespace
      metadata:
        name: dns-operator-system
```

Reply:
I do not think we can do that. 

Even if the manifests do not have the namespace object, 
```yaml
---
apiVersion: v1
kind: Namespace
metadata:
  labels:
    control-plane: controller-manager
  name: system
```

all the (namespaced) resources have the namespace set to `limitador-operator-system` (defined in config/default/kustomization.yaml)

Reply:
I got you know. You mean still setting the namespace to `{{ .Release.Namespace }}` and keep the namespace object in the `default` overlay. 

I will go for it.